packages/storybook/.storybook/preview-head.html

```diff
@@ -0,0 +1,78 @@
+<style>
+    .sbdocs.sbdocs-h1:first-of-type {
+        margin-top: 64px;
+    }
+
+    .sbdocs.sbdocs-h1 {
+        color: #000000;
+        font-size: 40px;
+        font-weight: 700;
+        font-family: "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
+        margin: 64px 0px 32px;
+        letter-spacing: -0.03em;
+    }
+
+    .sbdocs.sbdocs-h2:first-of-type {
+        margin-top: 48px;
+        padding-top: 48px;
+    }
+
+    .sbdocs.sbdocs-h2 {
+        color: #000000;
+        font-size: 24px;
+        font-weight: 600;
+        font-family: "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
+        margin: 48px 0px 24px;
+        padding: 48px 0px 0px;
+        border-top: 1px solid #EDEBE9;
+        border-bottom: none;
+        letter-spacing: -0.02em;
+    }
+
+    .sbdocs.sbdocs-h3:first-of-type {
+        margin-top: 32px;
+    }
+
+    .sbdocs.sbdocs-h3 {
+        color: #000000;
+        font-size: 16px;
+        font-weight: 600;
+        font-family: "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
+        margin: 32px 0px 12px;
+        letter-spacing: -0.02em;
+    }
+
+    .sbdocs.sbdocs-p {
+        font-size: 14px;
+        font-family: "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
+        line-height: 24px;
+        margin: 16px 0px;
+    }
+
+    pre {
+        background-color: #faf9f8 !important; /* we need to find a way to override the classes not made by storybook. no sbodcs tag for this one */
+    }
+
+    tr th {
+        background-color: #faf9f8;
+        text-align: left;
+    }
+
+    tr:nth-of-type(2n) {
+        background-color: #faf9f8;
+    }
+
+    .sbdocs.sbdocs-wrapper {
+        padding: 4rem 64px;
+    }
+
+    body {
```

Comment:
If we are assigning a font family to the body, is it not being applied to all the h1, h2, p blocks?

Reply:
the stylesheet provided by story book is overwriting it for the h1 h2 h3 blocks I believe, I should remove the body CSS I believe since I think its getting overridden.

Reply:
I looked at the styleset, I believe you can optimize it a bit :)

Reply:
okay will take a look! 😊